### PR TITLE
Added views and test to Comment

### PR DESCRIPTION
**Pull request by skramar**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVPYTP-39)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/comment/views.py [hunk 1]
```diff
@@ -1,7 +1,141 @@
...
+                        HttpRequest with error if parameters are bad
+
+        """
+        if comment_id:
+            comment = Comment.get_by_id(comment_id)
+            if comment:
+                comment = comment.to_dict()
+                return JsonResponse(comment, status=200)
+            return HttpResponse(status=404)
+        return HttpResponse(status=400)
```
Comment (redmyt) [inline eventually/comment/views.py]: improve logic without comment id

File: eventually/comment/views.py [hunk 2]
```diff
@@ -1,7 +1,141 @@
...
+
+        task = Task.get_by_id(task_id)
+        vote = Vote.get_by_id(vote_id)
+        event = Event.get_by_id(event_id)
+        team = Team.get_by_id(team_id)
+
+        if not (task or vote or event or team):
+            return HttpResponse(status=404)
+
+        data = comment_data_validator(request.body)
```
Comment (redmyt) [inline eventually/comment/views.py]: validator should returns `true` or  `false`

File: eventually/comment/views.py [hunk 3]
```diff
@@ -1,7 +1,141 @@
...
+        :type comment_id: int
+
+        :return: return JsonResponse within comment data with status 200 if parameters are good or
+                        HttpRequest with error if parameters are bad
+        """
+        data = comment_data_validator(request.body)
+        if not data:
+            return HttpResponse(status=400)
+
+        if comment_id:
```
Comment (redmyt) [inline eventually/comment/views.py]: unnecessary statement

File: eventually/comment/views.py [hunk 4]
```diff
@@ -1,7 +1,141 @@
...
+        :type request: HttpRequest
+
+        :param comment_id: id of Comment model
+        :type comment_id: int
+
+        :return: return JsonResponse within comment data with status 200 if parameters are good or
+                        HttpRequest with error if parameters are bad
+
+        """
+        if comment_id:
```
Comment (redmyt) [inline eventually/comment/views.py]: unnecessary statement

File: eventually/team/views.py [hunk 5]
```diff
@@ -9,3 +9,4 @@
 class TeamView(View):
     """Team view handles GET, POST, PUT, DELETE requests.
```
Comment (redmyt) [inline eventually/team/views.py]: fix docstring

File: eventually/tests/unittests/comment/testview.py [hunk 6]
```diff
@@ -0,0 +1,240 @@
+"""
+Comment Views Test.
```
Comment (redmyt) [inline eventually/tests/unittests/comment/testview.py:2]: add `===...` for name

File: eventually/tests/unittests/comment/testview.py [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+This module provides complete testing for all Comment's views functions.
+"""
+from django.test import TestCase, Client
+from authentication.models import CustomUser
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+from team.models import Team
+from comment.models import Comment
+from comment.views import CommentView
+import json
```
Comment (redmyt) [inline eventually/tests/unittests/comment/testview.py]: imports at first,`from` after imports.

File: eventually/tests/unittests/comment/testview.py (hunk 6 above)
Comment (skramar) [reply eventually/tests/unittests/comment/testview.py:2]: done

File: eventually/team/views.py (hunk 5 above)
Comment (skramar) [reply eventually/team/views.py]: done

File: eventually/comment/views.py (hunk 4 above)
Comment (skramar) [reply eventually/comment/views.py]: done

File: eventually/comment/views.py (hunk 3 above)
Comment (skramar) [reply eventually/comment/views.py]: done

File: eventually/comment/views.py (hunk 2 above)
Comment (skramar) [reply eventually/comment/views.py]: done

File: eventually/comment/views.py (hunk 1 above)
Comment (skramar) [reply eventually/comment/views.py]: done

**Pull request closed without merging**